src/main/kotlin/no/nav/syfo/behandlendeenhet/kafka/KafkaBehandlendeEnhetConfig.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.syfo.behandlendeenhet.kafka
+
+import no.nav.syfo.application.ApplicationEnvironmentKafka
+import no.nav.syfo.application.kafka.JacksonKafkaSerializer
+import no.nav.syfo.application.kafka.commonKafkaAivenProducerConfig
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.common.serialization.StringSerializer
+import java.util.*
+
+fun kafkaBehandlendeEnhetProducerConfig(
+    applicationEnvironmentKafka: ApplicationEnvironmentKafka,
+): Properties {
+    return Properties().apply {
+        putAll(commonKafkaAivenProducerConfig(applicationEnvironmentKafka))
+        this[ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG] = StringSerializer::class.java.canonicalName
+        this[ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG] = JacksonKafkaSerializer::class.java.canonicalName
```

Comment:
Funker dette eller må det lages en KBehandlendeEnhetUpdateSerializer som vi gjør i en del andre apper? Hadde vært nice å slippe 😄

Reply:
Jeg så på `KafkaBehandlerDialogmeldingConfig` i `isdialogmote`, der var det gjort sånn uten noen flere serializers 🤔 Men jeg ser at det er laget en egen for dialogmelding i samme repo. Det fungerer hvertfall å publisere på topicen i dev, vil tro det hadde kræsjer der dersom serializeringen ikke fungerer?

Reply:
Hvis det funker i dev er det sikkert good ja 👍